src/turing_model.jl

```diff
@@ -149,12 +205,58 @@ function turing_model(
     if standardize
         μ_X, σ_X, X = standardize_predictors(X)
         μ_y, σ_y, y = standardize_predictors(y)
+        if !isnothing(Z)
+            #TODO: implement random-effects slope
+            throw(
+                ArgumentError(
+                    "TuringGLM currently does not support random-effects for slope terms"
+                ),
+            )
```

Comment:
```suggestion
            msg = "TuringGLM currently does not support random-effects for slope terms"
            throw(ArgumentError(msg))
```
No one likes to read brackets at various indentation levels. Also, this takes 50% less lines